packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx

```diff
@@ -55,6 +70,27 @@ export const Appearance = () => {
         <Label htmlFor={filledDarkerId}>Filled darker appearance</Label>
         <Input appearance="filled-darker" id={filledDarkerId} />
       </div>
+    </>
+  );
+};
+
+export const Appearance = () => {
```

Comment:
We should add a test to test the override

Reply:
Agree, but it add to `FluentProvider` stories to show what it does

Reply:
Reverted the story, added a VR test for all affected components.
Do you think we want to add any documentation? I thought we would intentionally keep it undocumented (for now).

Reply:
> Do you think we want to add any documentation? I thought we would intentionally keep it undocumented (for now).

Let's keep it undocumented for now.